pwa/src/templates/componentResult/ComponentResultsTemplate.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from "react";
+import * as styles from "./ComponentResultsTemplate.module.css";
+import * as _ from "lodash";
+import { RichContentCard, Tag } from "@conduction/components";
+import { Card, Link } from "@gemeente-denhaag/components-react";
+import { HamburgerIcon, HouseIcon } from "@gemeente-denhaag/icons";
+import { Table, TableBody, TableCell, TableHeader, TableRow } from "@gemeente-denhaag/table";
+import { useTranslation } from "react-i18next";
+import { navigate } from "gatsby";
+import { ArrowRightIcon } from "@gemeente-denhaag/icons";
+
+interface ComponentsTemplateProps {
+  type: string;
```

Comment:
We could expand on this `type` prop, like so:


```
export type TComponentResultsLayout = "table" | "cards" | "layer";

interface ComponentsTemplateProps {
  type: TComponentResultsLayout;
  results: any[];
}
```

This way, when implementing this component, we cannot set a type (`string`) that does not exist.

Reply:
Note: we export the type so that we can import it where we're implementing this component, see other comments.